Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,16 @@
 import React from 'react';
 import './Header.css';
+import moment from 'moment';
```

Comment:
Your import here failed for me upon trying to look at this through codesandbox. I was able to make the dependency work on there, but you will want to investigate it further. I recommend double checking your package.json file.